src/components/emailLogin.vue

```diff
@@ -169,14 +169,18 @@
 		async mounted(){
```

Comment:
This file shuldn't be there
I don't how to delete it from the commit, it's need no to review, i'll push actual version later